examples/webpack/src/ui/custom-file-extension.customjsx

```diff
@@ -0,0 +1,27 @@
+import { styled } from '@compiled/react';
+
+export const CustomFileExtensionStyled = styled.div({
+  color: 'orange',
+  fontWeight: 500,
+
+  ':hover': {
+    color: 'red',
+  },
+
+  ':focus': {
+    color: 'blue',
+  },
+});
+
+export const CustomFileExtensionCss = {
```

Comment:
This should be lower case as `customFileExtensionCss`, same in the parcel example

I'd also use the explicit `css`

Reply:
Unfortunately haven't got around to investigating yet but it appears the parcel loader breaks when importing any `css(...)` calls, not just in this PR. Will look soon as we'll need to raise a bug.

Reply:
For later reference, the call stack is:
```🚨 Build failed.
@compiled/parcel-transformer: /Users/jlane2/Code/compiled/examples/parcel/src/app.js: Cannot read property 'buildError' of undefined
TypeError: /Users/jlane2/Code/compiled/examples/parcel/src/app.js: Cannot read property 'buildError' of undefined
    at NodePath.buildCodeFrameError (/Users/jlane2/Code/compiled/node_modules/@babel/traverse/lib/path/index.js:139:21)
    at Object.buildCodeFrameError (/Users/jlane2/Code/compiled/packages/babel-plugin/dist/utils/ast.js:59:52)
    at Object.buildCss (/Users/jlane2/Code/compiled/packages/babel-plugin/dist/utils/css-builders.js:621:17)
    at Object.buildCss (/Users/jlane2/Code/compiled/packages/babel-plugin/dist/utils/css-builders.js:567:30)
    at Object.visitCssPropPath (/Users/jlane2/Code/compiled/packages/babel-plugin/dist/css-prop/index.js:75:36)
    at PluginPass.JSXOpeningElement (/Users/jlane2/Code/compiled/packages/babel-plugin/dist/babel-plugin.js:197:28)
    at newFn (/Users/jlane2/Code/compiled/node_modules/@babel/traverse/lib/visitors.js:177:21)
    at NodePath._call (/Users/jlane2/Code/compiled/node_modules/@babel/traverse/lib/path/context.js:53:20)
    at NodePath.call (/Users/jlane2/Code/compiled/node_modules/@babel/traverse/lib/path/context.js:40:17)
    at NodePath.visit (/Users/jlane2/Code/compiled/node_modules/@babel/traverse/lib/path/context.js:100:31)
/Users/jlane2/Code/compiled/examples/parcel/src/app.js
```